src/docs2vecs/subcommands/indexer/document/chunk.py

```diff
@@ -4,6 +4,7 @@ def FromDict(dict):
         chunk = Chunk()
         chunk.document_id = dict["document_id"]
```

Comment:
nit: (not for you @OrianeLanfranchi - a general remark)

We should avoid using ambiguous names for the input parameters. In this case `dict` means an instance of an object of a builtin type `dict` I suppose.. but python is dynamically typed, so it could be anything in practice.